scripts/acs-cmd/acs-cmd-list-resources.js

```diff
@@ -1,46 +0,0 @@
-#!/usr/bin/env node
```

Comment:
We still need resources right?

Reply:
Done in https://github.com/magda-io/magda/pull/2432